queue/src/fiskaltrust.Middleware.Storage.SQLite/SQLiteStorageBootstrapper.cs

```diff
@@ -32,13 +33,19 @@ public class SQLiteStorageBootstrapper : BaseStorageBootStrapper
         private readonly Guid _queueId;
         private readonly SQLiteStorageConfiguration _sqliteStorageConfiguration;
         private readonly ILogger<IMiddlewareBootstrapper> _logger;
+        private readonly IJournalFRCopyPayloadRepository _journalFRCopyPayloadRepository;
 
-        public SQLiteStorageBootstrapper(Guid queueId, Dictionary<string, object> configuration, SQLiteStorageConfiguration sqliteStorageConfiguration, ILogger<IMiddlewareBootstrapper> logger)
+        public SQLiteStorageBootstrapper(Guid queueId, Dictionary<string, object> configuration,
+            SQLiteStorageConfiguration sqliteStorageConfiguration, ILogger<IMiddlewareBootstrapper> logger,
+            SQLiteJournalFRRepository journalFRRepository,
+            IJournalFRCopyPayloadRepository journalFRCopyPayloadRepository)
+            : base(journalFRRepository, journalFRCopyPayloadRepository) 
         {
             _configuration = configuration;
             _sqliteStorageConfiguration = sqliteStorageConfiguration;
             _queueId = queueId;
             _logger = logger;
+            _journalFRCopyPayloadRepository = journalFRCopyPayloadRepository;
         }
```

Comment:
```suggestion
        private readonly IJournalFRCopyPayloadRepository _journalFRCopyPayloadRepository;

        public SQLiteStorageBootstrapper(Guid queueId, Dictionary<string, object> configuration,
            SQLiteStorageConfiguration sqliteStorageConfiguration, ILogger<IMiddlewareBootstrapper> logger)
            : base(journalFRRepository, journalFRCopyPayloadRepository) 
        {
            _configuration = configuration;
            _sqliteStorageConfiguration = sqliteStorageConfiguration;
            _queueId = queueId;
            _logger = logger;
        }
```

If you look at the [Queue.SQLite Bootstrapper](../fiskaltrust.Middleware.Queue.SQLite/PosBootstrapper.cs) the `SQLiteStorageBootStrapper` is directly instantiated with `new SQLiteStorageBootStrapper(....)` the repositories don't exist there as it is the job of the `SQLiteStorageBootStrapper` to create them.

You need to create instances of these two repositories like below with the configurationRepository `_configurationRepository = new SQLiteConfigurationRepository(_connectionFactory, _sqliteFile);`

and pass those to the methods of the base storage like done in the `PersistConfigurationAsync` call below `await PersistConfigurationAsync(baseStorageConfig, _configurationRepository, logger).ConfigureAwait(false);` (the `_configurationRepository` is passed in here)